apiserver/ServeSubmission.md

```diff
@@ -0,0 +1,681 @@
+                  }
+               },
+               {
+                  "name":"create_order",
+                  "numReplicas":1,
+                  "rayActorOptions":{
+                     
+                  }
+               },
+               {
+                  "name":"DAGDriver",
+                  "numReplicas":1,
+                  "rayActorOptions":{
+                     
+                  }
+               }
+            ]
+         }
+      }
+   },
+   "proxies":{
+      "4def7ebc8f2307aeb166f3a9585c7d7ed0d974506cc6d764d812b2ba":{
+         "nodeId":"4def7ebc8f2307aeb166f3a9585c7d7ed0d974506cc6d764d812b2ba",
+         "nodeIp":"10.244.2.3",
+         "actorId":"985d4e1b1115bfd8aadfcd5201000000",
+         "actorName":"SERVE_CONTROLLER_ACTOR:SERVE_PROXY_ACTOR-4def7ebc8f2307aeb166f3a9585c7d7ed0d974506cc6d764d812b2ba",
+         "workerId":"3ca22e41adcba21db3f155c344945306c600a0db994f3e8d9349c523",
+         "status":"HEALTHY",
+         "logFilePath":"/serve/proxy_10.244.2.3.log"
+      },
+      "e986907a0656721d0870c9797ac752edf97ea401599e76ababa68e26":{
+         "nodeId":"e986907a0656721d0870c9797ac752edf97ea401599e76ababa68e26",
+         "nodeIp":"10.244.1.2",
+         "actorId":"a6a37816bca13d25d88a900201000000",
+         "actorName":"SERVE_CONTROLLER_ACTOR:SERVE_PROXY_ACTOR-e986907a0656721d0870c9797ac752edf97ea401599e76ababa68e26",
+         "workerId":"d8ea829af0423c1dcf640d4531c284ab5b2f74b6ec764a6e75d64f98",
+         "status":"HEALTHY",
+         "logFilePath":"/serve/proxy_10.244.1.2.log"
+      }
+   }
+}
+```
+
+### Delete Serve applications
+
+Finally, you can delete serve applications using the following command:
+
+```shell
+curl -X DELETE 'localhost:31888/apis/v1/namespaces/default/serveapplication/test-cluster' \
+--header 'Content-Type: application/json' 
+```
+
+You can validate job deletion by looking at the Ray dashboard (serve pane) and ensuring that it was removed
+
+## Managing RayServe with the API server vs RayService CRD
+
+[RayService CRD](https://docs.ray.io/en/latest/cluster/kubernetes/getting-started/rayservice-quick-start.html#kuberay-rayservice-quickstart) provides many important features, including:
+
+* In-place updating for Ray Serve applications: See RayService for more details.
+* Zero downtime upgrading for Ray clusters: See RayService for more details.
+* High-availabilable services: See [RayCluter high availability](HACluster.md) for more details.
+
+So why this implementation? Several reasons:
+
+* It is more convinient in development. You can create a cluster and then deploy/undeploy applications until you are happy with results.
+* You can create Ray cluster for serve with the set of features that you want, including [high availabilty](HACluster.md), [autoscaling support](Autoscaling.md), etc. You can choose cluster configuration differently for testing vs production. Moreover, all of this can be done using [Python](../clients/python-apiserver-client/python_apiserver_client)
```

Comment:
```suggestion
* You can create Ray cluster for serve with the set of features that you want, including [high availability](HACluster.md), [autoscaling support](Autoscaling.md), etc. You can choose cluster configuration differently for testing vs production. Moreover, all of this can be done using [Python](../clients/python-apiserver-client/python_apiserver_client)
```

Reply:
done